### ISSUE TITLE
Order of structure of symbols

### ISSUE THREAD

**Issue by christiankral** [otfried/ipe#246]
When selecting the group of symbols I get a different sort order of the groups each time I restart IPE. 

![image](https://user-images.githubusercontent.com/4184218/57522529-1aab6300-7323-11e9-91d2-5f39b1fd9eee.png)

![image](https://user-images.githubusercontent.com/4184218/57522752-4b8b9800-7323-11e9-8840-5dc7aa7bc2c3.png)

The order of symbols within the group appears always in the same way, however. 

This issue is related with #237 and version 7.2.12 of IPE (Linux 64bit Ubuntu 16.04 version)

**Comment by otfried** [comment]
Indeed, the Lua map is a hash table and is not deterministic.   

In any case, one should probably sort the group names alphabetically.   I'll fix it in the next release.   You can do it yourself by inserting the `table.sort` call in the function `select_symbol` in `symbols.lua`, like this:
```
  local groups = {}
  for g,sbg in pairs(symbolsByGroup) do
    groups[#groups+1] = g
  end
  table.sort(groups)
  local d = ipeui.Dialog(model.ui:win(), "Use symbol")
  groups.action = function (d)
...
```

**Comment by otfried** [comment]
Fixed in 7.2.13.
